pkg/virt-controller/watch/volume-migration/volume-migration_test.go

```diff
@@ -81,11 +81,6 @@ var _ = Describe("Volume Migration", func() {
 			), libvmi.NewVirtualMachine(libvmi.New(
 				libvmi.WithPersistentVolumeClaim("disk0", "vol2"), withFilesystemVolume("disk1", "vol4"),
 			)), fmt.Errorf("invalid volumes to update with migration: filesystems: [disk1]")),
-			Entry("with an invalid hotplugged volume", libvmi.New(
-				libvmi.WithPersistentVolumeClaim("disk0", "vol0"), withFilesystemVolume("disk1", "vol1"),
-			), libvmi.NewVirtualMachine(libvmi.New(
-				libvmi.WithPersistentVolumeClaim("disk0", "vol2"), withHotpluggedVolume("disk1", "vol4"),
-			)), fmt.Errorf("invalid volumes to update with migration: hotplugged: [disk1]")),
 		)
```

Comment:
Should there be a positive hotplug volume test?

Reply:
You are right, I made the entry as a valid option